src/main/java/org/opensearch/flowframework/rest/RestCreateWorkflowAction.java

```diff
@@ -84,8 +85,10 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
         String workflowId = request.param(WORKFLOW_ID);
         String[] validation = request.paramAsStringArray(VALIDATION, new String[] { "all" });
         boolean provision = request.paramAsBoolean(PROVISION_WORKFLOW, false);
+        boolean reprovision = request.paramAsBoolean(REPROVISION_WORKFLOW, false);
```

Comment:
We should a separate Reprovision handler like Deprovisioning

Reply:
I will leave it as is : https://github.com/opensearch-project/flow-framework/pull/804#issuecomment-2266133780